swri_route_util/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(swri_route_util)
 set(BUILD_DEPS 
   marti_nav_msgs
```

Comment:
It looks like you're missing dependencies on `marti_common_msgs`, `swri_route_util`, and `visualization_msgs`.